coordinator/src/main/scala/filodb.coordinator/queryplanner/DefaultPlanner.scala

```diff
@@ -459,4 +486,18 @@ object PlannerUtil extends StrictLogging {
     childTargets.iterator.drop(rnd.nextInt(childTargets.size)).next
    }
   }
+
+  /**
+   * Returns the set of shards that contain an ExecPlan's data.
+   */
+   def getShardSpanFromEp(ep: ExecPlan): Set[Int] = {
```

Comment:
It was used to order children of an ExecPlan during `printTree`, but it's now removed (along with the new `printTree` parameter) since the print order is no longer random.